Source/WebCore/Modules/identity/IdentityRequestProvider.idl

```diff
@@ -23,7 +23,9 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+typedef (OpenID4VPRequest) DigitalCredentialRequest;
```

Comment:
Why the parentheses?

Reply:
this was for `(X or Y)`... which I'll get rid of.